Project-1/Praoject_1_final_version

```diff
@@ -0,0 +1,384 @@
+  	T(x) := ∃z.(P(z,x) ∧ ∃u.(P(z,u) ∧ ∃w.(P(z,w) ∧ ¬F(u) ∧ ¬F(w) ∧ ∀n.((P(z,n) ∧ ¬F(n)) → (n=x V n=u V n=w)) x≠u≠w))
+
+  ```
+
+4. Let V be a vocabulary of the attribute (concept) language with complements (ALC) consisting of a role name "parent_of" and a concept name "Male". Interpret parent_of as "x is a parent of y" and M as "x is male". Where possible define the following formulas in this vocabulary; where not possible, explain why: 
+  ```
+  (a)  B that says that x is a brother of y
+
+    person := M ⊔ ¬M ⊔ ¬(M ⊓ ¬M)
+
+    P2 (parent of at least 2) := (∃parent_of.person ≥ 2) ⊓ ¬(∃parent_of.person = 1)
+
+    B (brother) := M ⊓ ∃P2¯.person
+  
+  (b)  A that says that x is an aunt of y
+
+    S (sister) := ¬M ⊓ ∃P2¯.person
+
+    Sib (sibiling) := B ⊔ S
+
+  	A (aunt) := ¬M ⊓ (∃P2¯.((parent_of.person ≥ 2).(parent_of.person)) ⊔ ¬parent_of.person)
+
+    U (Uncle) := M ⊓ (∃P2¯.((parent_of.persom ≥ 2).(parent_of.person)) ⊔ ¬parent_of.person)
+  	
+  (c)  C that says that x and y are cousins
+  	
+    This cannot be, since in DL only one entity or role can be described at once. Thus, we cannot say the statement x and y are cousins. 
+  	
+  (d)  O that says that x is an only child  
+
+    O := 	¬∃p2¯.person
+	
+  (e)  T that says that x has exactly two brothers 
+
+  This formula cannot be defined in the vocabulary of ALC with only the role name "parent_of" and the concept name "Male". ALC only allows expressing subsumption relationships between concepts, not cardinality or other relationships between roles. To express the relationship "x has exactly two brothers," we would need to have a role name for "brother_of" or similar, as well as a way to express cardinality constraints.
+  	
+  ```
+
+5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+  ```
+
+  T-Box:
+  {
+    B ⊑ Person
+    B ⊑ M
+    M ⊑ Person
+    A ⊑ ¬M
+    A ⊑ Person
+    }
+  
+  A-Box:
+  { 
+    John : M
+    John : parent_of.Chris
+    John : B
+    Chris : M
+    Mary : ¬M
+    Mary: A
+    Alex : parent_of.Mary
+    Alex : parent_of.John
+    }
+
+  ΔI = {John, Mary, Alex, Chris}
+
+  Named Individuals:
+  JohnI = J,
+  MaryI = M,
+  AlexI = A,
+  Chris = C
+
+  Concept Assignments:
+  PersonI = {J, M, A, C},
+  MI = {J, A, C},
+  FI = {M},
+  BI = {J},
+  AI = {M}
+
+Role Assignments:
+parentofI = {(J, C), (A, J), (A, M)}
+
+  ```
+6. Explain the difference - using natural language - between the first-order prefixes:
+  ```
+  (a) ∃x∀y and ∀x∃y
+	For statements coming after ∃x∀y, there must be an x included in them while there is no necessity of them having a y. Also, for such statements to be true, they must be true in all conditions of y (T or F), but as long as they are true for at least one case of x they are fine.
```

Comment:
I do not understand this. What do you mean by "statements" coming after ∃x∀y? Wffs? Predicates? Relations? 
What do you mean by x must be included in these statements but there's no necessity of them having a y? Are you saying the existential quantifier always binds but the universal doesn't?